pkg/virt-operator/webhooks/kubevirt-update-admitter.go

```diff
@@ -39,16 +39,17 @@ type KubeVirtUpdateAdmitter struct {
 	Client kubecli.KubevirtClient
 }
 
-// NewKubeVirtUpdateAdmitter creates a KubeVirtUpdateAdmitter
-func NewKubeVirtUpdateAdmitter(client kubecli.KubevirtClient) *KubeVirtUpdateAdmitter {
+// NewKubeVirtAdmitter creates a KubeVirtUpdateAdmitter
+func NewKubeVirtAdmitter(client kubecli.KubevirtClient) *KubeVirtUpdateAdmitter {
```

Comment:
Why did you drop the "Update" part? It does create a new `KubeVirtUpdateAdmitter`. At least, if you change one, you should change the other, no?

Reply:
Thought I did a replace all. I have updated `KubeVirtUpdateAdmitter` to be `KubevirtAdmitter`